app/views/events/index.html.haml

```diff
@@ -1,15 +1,17 @@
 %section
   .page-header
     .pull-right
-      - if policy(@conference).manage?
+      - if policy(@conference).manage? and !showing_my_events
         = action_button "primary", t('events_module.add_event'), new_event_path, hint: t('events_module.add_event_hint')
```

Comment:
Why wouldn't you allow adding new events in the "my" view?

Reply:
I was keeping legacy behavior. The "My Events" page didn't have this button. 

I will remove the condition. i.e., I will add the "Add Event" button to "My Events" page.